app/native/components/HelloWorld.js

```diff
@@ -1,5 +1,5 @@
-import React, {
-  Component,
+import React, { Component } from 'react';
+import {
```

Comment:
Btw, i tried the changes you added and I got it to build minus the proptypes stuff. Would be good if this PR goes in.